src/app/Fake.Runtime/FakeRuntime.fs

```diff
@@ -205,21 +170,31 @@ let paketCachingProvider (config:FakeConfig) cacheDir (paketApi:Paket.Dependenci
           installModel.GetRuntimeLibraries graph rid targetProfile
           |> Seq.map (fun fi -> DependencyFile.Library { File = fi.Library.Path })
           |> Seq.toList
+        // please see: https://docs.microsoft.com/en-us/dotnet/core/rid-catalog
+        // some native libraries, such as Sqlite.Interop are distributed without version specific.
+        // THe runtime identifier on windows 10 machine will be win10-x64 however, Sqlite.Interop
+        // will have an RID of win-x64
+        let runtimeLibrariesNotVersionSpecific =
+          installModel.GetRuntimeLibraries graph ridNotVersionSpecific targetProfile
+          |> Seq.map (fun fi -> DependencyFile.Library { File = fi.Library.Path })
+          |> Seq.toList
+
```

Comment:
I'm not sure if this is how it is supposed to work, I'd assume some libraries might still run into issues.
Basically the runtime identifiers are organized like a tree (https://docs.microsoft.com/en-us/dotnet/core/rid-catalog), last time I checked those RIDs where pushed with a NuGet package (you should find the logic in the Paket.Core code. This might have changed by now.
But basically we should know that a win-x64 is compatible on a win10-x64 runtime and take that (disclaimer: I wrote that code in Paket and it might have changed by now or is no longer working)

Of course we can release it as-is and see if someone complains.

Reply:
For the native assemblies not sure why it breaks, but the two changes I think affect it are:
1. Either `Paket.Core`, in which we updated to a major version, so it may have some breaking changes in that area.
2. Or the removal of `Microsoft.DotNet.PlatformAbstractions` since it is now deprecated by Microsoft and using built-in runtime identifier property.

As you pointed out it should resolve the native package since it is a graph and `win10-x64` is a child of `win-x64`. When I debug it, the `InstallModel` resolved it with `win10-x64` but the runtime identifier from SDK was `win-x64`, and `InstallModel` didn't return anything in that configuration.

I'm not familiar with this area, if you have any suggestions I'm all open.

Reply:
If you want to take a look, I'd say that if you debugged it to `InstallModel` it is most likely a Paket bug, so the easiest would be to add a unit-test there and try to solve it there and then ask @forki to make a release :)  
IIRC there are already tests for the runtime identifiers there, so it shouldn't be too hard to add this scenario (but that is always easy to write, I could easily be wrong on that)